src/ec/dasharo/ec/Kconfig

```diff
@@ -29,3 +29,7 @@ config EC_DASHARO_EC_UPDATE_FILE
 config EC_DASHARO_EC_FLASH_SIZE
 	hex
 	default 0x20000
+
+config EC_DASHARO_EC_CPU_FAN_COUNT
+	int
+	default 1
```

Comment:
Shouldn't be a range 1,2?

Reply:
Done in https://github.com/Dasharo/coreboot/pull/646/commits/d7ffd78a91e4c2a3dc93eb78e238b2c381723c81